helm/hpcc/values.schema.json

```diff
@@ -421,7 +421,18 @@
         "defaultSprayParts" : {
             "description": "Number of parts sprayed by default",
             "type" : "integer"
+        },
+        "cost" : {
+            "description": "Costs associated with the storage and use of the plane",
+            "type" : "object",
+            "properties": {
+                "storageAtRest" : {
+                    "description": "Storage cost (GiB/month)",
```

Comment:
is per month sensible granularity for us, e.g. why not per day?

Reply:
The pricing tables provided by the cloud providers are on a monthly basis.  I thought it would make it easier to setup and easier to identify mistakes if the time period was the same.

Reply:
that's fine - I didn't realize all cloud providers used monthly basis.

Reply:
@ghalliday (re default values) I haven't been able to provide a default cost value in the values.yaml because planes are not defined in the default values.yaml.  I've used the "default" attribute in the schema but that was ignored.